docs/index.md

```diff
@@ -1008,7 +1008,7 @@ Some assertion libraries will supply their own diffs, in which case Mocha's will
 
 Mocha's own diff output does not conform to any known standards, and is designed to be human-readable.
 
-> _New in v9.2.1_
+> _Since in v9.2.1_
```

Comment:
@TG199 did you mean to get this one too? 🙂

Reply:
yikes, missed it.